packages/react-ui-core/src/Highlighter/Highlighter.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import themed from 'react-themed'
+import classnames from 'classnames'
+import { randomId } from '@rentpath/react-ui-utils'
+
+@themed(/^Highlighter/)
+
+export default class Highlighter extends Component {
+  static propTypes = {
+    pattern: PropTypes.string,
+    children: PropTypes.string,
+    theme: PropTypes.object,
+    indexHighlighted: PropTypes.number,
+  }
+
+  static defaultProps = {
+    text: '',
+    children: '',
+    theme: {},
+  }
+
+  get indexHighlighted() {
+    return this.props.indexHighlighted + 1
+  }
+
+  renderHighlightedText() {
+    const { pattern, children } = this.props
+
+    const segments = children.split(pattern)
+
+    return segments.reduce((prev, curr, index) => [prev,
+      this.renderHighlightedSegment(index),
+      curr])
+  }
+
+  renderHighlightedSegment(index) {
+    const { pattern, theme, indexHighlighted } = this.props
+
+    if (indexHighlighted === undefined || index === indexHighlighted) {
+      return (
+        <span
+          data-tid="Highlighter-match"
+          className={classnames(theme.Highlighter_Match)}
+          key={randomId()}
+        >
+          {pattern}
+        </span>
+      )
+    }
+    return pattern
+  }
+
+  render() {
+    const { theme } = this.props
+
+    return (
+      <div
+        className={classnames(theme.Highlighter)}
```

Comment:
No need for `classnames` here as you only have one class. I'm guessing you're missing `className` prop here which means you'd need `classnames` after all 😄